sqlitecluster/SQLite.cpp

```diff
@@ -385,13 +385,13 @@ bool SQLite::beginTransaction(TRANSACTION_TYPE type) {
     return _insideTransaction;
 }
 
-bool SQLite::verifyTable(const string& tableName, const string& sql, bool& created) {
+bool SQLite::verifyTable(const string& tableName, const string& sql, bool& created, const string& type) {
```

Comment:
Should we maybe create a verifyView instead? We have verifyIndex as well, so I think that keeping the types separate would be good. Then separate the code into a third abstract method that received the type. To simplify the call..

Reply:
verifyIndex has specific code for indexes, but in this case views and tables are the same and the code would be identical. I could create a `verifyView` and leave `verifyTable` and both could call another method but then what would method be called? And is it really better than having this?

Reply:
I feel like for consistency, it would be better to have specific methods. It's kind of confusing to say `verifyTable of type view`. But then, views are indeed virtual tables, so maybe we don't really care about that?

Reply:
I won't block on this, since I think it's not super important, it's just a matter of definition. If we consider that virtual tables are the same as tables, verifyTable type view is not a problem. Approving the PR

Reply:
Cool, I'll take it because views are indeed tables and because I would not know how to name the methods otherwise 😄